package.json

```diff
@@ -30,23 +30,27 @@
     "build:esm": "tsc --noEmit false --outDir ./_esm --sourceMap",
     "build:types": "tsc --module esnext --declarationDir ./_types --emitDeclarationOnly --declaration --declarationMap",
     "clean": "rm -rf ./_esm ./_cjs ./_types",
-    "deps:check": "knip",
+    "deps:check": "knip --production --tags=-knipignore",
     "format:check": "prettier --check .",
     "lint": "eslint --cache .",
+    "prepare": "husky",
     "prepublishOnly": "npm run build",
+    "test": "vitest run",
     "tsc": "tsc --noEmit"
   },
   "devDependencies": {
-    "@commitlint/cli": "^19.5.0",
+    "@commitlint/cli": "^19.6.1",
     "better-sort-package-json": "^1.1.0",
     "commitlint-config-bloq": "^1.0.0",
     "eslint": "^8.57.1",
-    "eslint-config-bloq": "^4.2.0",
-    "husky": "^4.3.8",
-    "knip": "^5.34.0",
-    "lint-staged": "^15.2.7",
-    "prettier": "^3.3.2",
-    "typescript": "^5.5.3"
+    "eslint-config-bloq": "^4.4.1",
+    "husky": "^9.1.7",
+    "knip": "^5.41.1",
+    "lint-staged": "^15.3.0",
+    "prettier": "^3.4.2",
+    "typescript": "^5.7.2",
+    "viem": "2.7.19",
+    "vitest": "2.1.8"
```

Comment:
Should we use a "^" range here too?

Reply:
I think in general I prefer to have fixed  versions. `viem` at least updates types (Which can make the build fail) between minor version updates 🤷🏽  so I'd keep this as is.

Reply:
Not happy with this but if the developers of those dependencies do not follow semver, even for types, we need to fix the versions.